src/cmd/singularity/cli/actions.go

```diff
@@ -36,6 +36,7 @@ func init() {
 		ExecCmd,
 		ShellCmd,
 		RunCmd,
+		testCmd,
```

Comment:
Every other `cobra.Command` struct is capitalized. Either they're all exported, or all not exported. I would say, capitalize this

Reply:
ok